### ISSUE TITLE
When just release, running cargo run --release getting error: Recipe release failed on line 14 with exit code 101

### ISSUE THREAD

**Issue by wksantiago** [HarborWallet/harbor#81]
When running cargo run --release 
    Finished release profile [optimized] target(s) in 0.24s
     Running target/release/harbor
getting Library libxkbcommon-x11.so could not be loaded.
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace
error: Recipe release failed on line 14 with exit code 101

**Comment by TonyGiorgio** [comment]
is this running in the nix shell?

**Comment by wksantiago** [comment]
yes

**Comment by TonyGiorgio** [comment]
what os?

**Comment by wksantiago** [comment]
Linux Mint 21.3 KDE Plasma V 5.24.7

**Comment by TonyGiorgio** [comment]
Okay thanks. Unfortunately it's really hard for us to hit full compilation coverage on every environment right out of the gate. There has been many linux issues across many distro's and linux GUI frameworks.

Hopefully as more linux devs start playing with and fixing these issues, they will get resolved. Hard for us to fix them without these devices.

**Comment by wksantiago** [comment]
Cool, thanks for the update and keep up the great work.  Will stay pending for developments.

**Comment by echennells** [comment]
@wksantiago I'm trying to reproduce but I'm not certain which version of Mint you are running,  is it the cinnamon edition?  I don't see any of them that use KDE by default.

**Comment by wksantiago** [comment]
its X11

**Comment by masud-abdulkadir** [comment]
yeah im running into the same issue, looking around for solutions. I've tried using the flake.nix here to no avail. @TonyGiorgio are you running NixOs?

**Comment by futurepaul** [comment]
the flake has been updated a lot since this was reported, can anyone re-test? I've tested on ubuntu / popos with wayland and x11 so those theoretically should work. haven't done anything kde tho
